tools/docker/builder/openwrt/build.sh

```diff
@@ -68,7 +68,7 @@ main() {
 
     eval set -- "$OPTS"
 
-    SUPPORTED_TARGETS="turris-omnia glinet-b1300"
+    SUPPORTED_TARGETS="turris-omnia glinet-b1300 netgear-rax40"
```

Comment:
So how did it work till now?

Reply:
This one is only used to inform the user of the supported targets, it's not actually used in CI.